templates/carousel.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="general-container">
+  <div class="carousel">
+      <div class="carousel-container">
+          <div class="card-wrapper">
+            {{#each carouselSlides}}
+              {{{module this}}}
+            {{/each}}
+          </div>
+          <div class="card-control">
+            <div class="control-container">
+              <div class="control__arrow-back arrow">
+                <img src="/images/carousel_arrow.svg" alt="">
```

Comment:
Please add in the alt attribute the value "Siguiente o Anterior" segun corresponda